doc/source/f2py/buildtools/index.rst

```diff
@@ -97,6 +80,12 @@ In theory keeping the above requirements in hand, any build system can be
 adapted to generate ``f2py`` extension modules. Here we will cover a subset of
 the more popular systems.
 
+.. note::
+
+   The current default build system for ``f2py`` is ``meson``.
+   If you need information about the migration from ``numpy.distutils`` to
+   ``meson``, refer to this section in the Version 1.26 documentation.
+
```

Comment:
I think we should have a link here to the document in question. Also, possibly in scope for this PR is that we still have `distutils` as the default if the Python version is less than `3.12` (see `f2py2e.py` around 556).

Reply:
In fact, perhaps until 3.12 becomes the lowest supported version we should keep `distutils-to-meson.rst` since (most) F2PY users probably haven't transitioned.